app/models/notice.rb

```diff
@@ -122,8 +122,7 @@ def language_enum
 
   acts_as_taggable_on :tags, :jurisdictions
 
-  accepts_nested_attributes_for :file_uploads,
-    reject_if: ->(attributes) { attributes['file'].blank? }
```

Comment:
`reject_if: ->(attributes) { [attributes['file'], attributes[:pdf_request_fulfilled]].all?(&:blank?) }`

So stop rejecting if another piece of relevant data comes in. Just spitballing here.